deploy/.env.template

```
@@ -22,5 +22,8 @@ DB_PASSWORD=
 REDIS_HOST=
 REDIS_PORT=6379
 
-# Config Server 주소(config-server 수정 시 삭제 예정)
-CONFIG_SERVER=
\ No newline at end of file
+LOKI_URL=
+ZIPKIN_ENDPOINT=
+
+# 예: 도메인 서비스명을 사용 -> /logs/user-service.log
+LOG_FILE_PATH=/logs/<도메인_서비스명>.log
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)